include/rocksdb/c.h

```diff
@@ -662,8 +658,8 @@ extern ROCKSDB_LIBRARY_API void rocksdb_options_set_source_compaction_factor(
     rocksdb_options_t*, int);
 extern ROCKSDB_LIBRARY_API void rocksdb_options_prepare_for_bulk_load(
     rocksdb_options_t*);
-extern ROCKSDB_LIBRARY_API void rocksdb_options_set_memtable_vector_rep(
```

Comment:
is this one still needed? https://github.com/facebook/rocksdb/blob/master/db/c.cc#L1780